packages/wonder-blocks-popover/src/components/__docs__/popover.stories.js

```diff
@@ -33,8 +58,276 @@ const styles = StyleSheet.create({
+    const [opened, setOpened] = React.useState(true);
+    return (
+        <View style={styles.row}>
+            <Popover
+                opened={opened}
+                onClose={() => {
+                    setOpened(false);
+                }}
+                content={({close}) => (
+                    <PopoverContent
+                        title="Controlled popover"
+                        content="This popover is controlled programatically. This means that is only displayed using the `opened` prop."
+                        actions={
+                            <Button
+                                onClick={() => {
+                                    close();
+                                }}
+                            >
+                                Click to close the popover
+                            </Button>
+                        }
+                    />
+                )}
+            >
+                <Button
+                    // eslint-disable-next-line no-console
+                    onClick={() => console.log("This is a controlled popover.")}
+                >
+                    Anchor element (it does not open the popover)
+                </Button>
+            </Popover>
+            <Strut size={Spacing.xLarge_32} />
+            <Button onClick={() => setOpened(true)}>
+                Outside button (click here to re-open the popover)
+            </Button>
+        </View>
+    );
+};
+
+Controlled.args = {
+    children: ({open}) => <Button onClick={open}>Trigger element</Button>,
+    content: (
+        <PopoverContent
+            closeButtonVisible
+            title="Title"
+            content="The popover content."
+            image={
+                <img
+                    src="/illustration.svg"
+                    alt="An illustration of a person skating on a pencil"
+                    width={288}
+                    height={200}
+                />
+            }
+        />
+    ),
+
+    dismissEnabled: true,
+};
+
+Controlled.parameters = {
+    docs: {
+        description: {
+            story:
+                `Sometimes you'll want to trigger a popover programmatically.
+                This can be done by setting the \`opened\` prop to \`true\`. In
+                this situation the \`Popover\` is a controlled component. The
+                parent is responsible for managing the opening/closing of the
+                popover when using this prop. This means that you'll also have
+                to update \`opened\` to \`false\` in response to the
+                \`onClose\` callback being triggered.\n\n` +
+                `Here you can see as well how the focus is managed when a
+                popover is opened. To see more details, please check the
+                **Accesibility section**.`,
+        },
+    },
+};
+
+/**
+ * With Actions
+ */
+export const WithActions: StoryComponentType = Template.bind({});
```

Comment:
The "show code" section for the WithActions story doesn't show the `actions` prop in the code.

![image](https://user-images.githubusercontent.com/13231763/187782648-9bc436e9-e78c-4a89-a202-078b3ac33b4c.png)

Reply:
@nishasy I pushed a new commit with your suggestions. I did as much as I could to keep the snippets useful :). Let me know what you think.

https://5e1bf4b385e3fb0020b7073c-lbbnucutzz.chromatic.com/?path=/docs/popover-popover--with-actions